### PR TITLE
Update api.mdx

### PR DESCRIPTION
**Pull request by ahwreck**
Add `Get a trip` to Trip Tracking API docs

### CODE REVIEW

File: docs/api.mdx [hunk 1]
```diff
@@ -1296,6 +1296,57 @@ A trip is a sequence of location updates with metadata and a unique ID, optional
...
+###### Sample request
+
+```bash
+curl "https://api.radar.io/v1/trips/299" \
+  -H "Authorization: prj_live_sk_..."
+```
+
+###### Authentication level
+
+Publishable
```
Comment (kochis) [inline docs/api.mdx:1320]: I think this is `secret` auth

File: docs/api.mdx [hunk 2]
```diff
@@ -1296,6 +1296,57 @@ A trip is a sequence of location updates with metadata and a unique ID, optional
   ]
 }
 ```
+#### Get a trip
+
+Gets a trip. The trip can be referenced by Radar `_id` or `externalId`.
+
+###### Definition
+
+<span className="badge badge--warning">GET</span> <code>https://api.radar.io/v1/trips/:id</code>
```
Comment (nickpatrick) [inline docs/api.mdx:1305]: @ahwreck Nice!

`GET` should be green, so `badge--success` instead of `badge--warning`

File: docs/api.mdx (hunk 1 above)
Comment (nickpatrick) [reply docs/api.mdx:1320]: +1

Comment (ahwreck) [reply docs/api.mdx:1320]: if it's secret we should update the PATCH trips to secret as well

File: docs/api.mdx (hunk 2 above)
Comment (ahwreck) [reply docs/api.mdx:1305]: gtk 👍

File: docs/api.mdx (hunk 1 above)
Comment (ahwreck) [reply docs/api.mdx:1320]: did another pull for the PATCH authentication

Comment (jsani-radar) [reply docs/api.mdx:1320]: @ahwreck - did you want to change this to secret?

File: docs/api.mdx (hunk 2 above)
Comment (jsani-radar) [reply docs/api.mdx:1305]: @ahwreck - did you want to change the `className` on this?

**Review by jsani-radar — APPROVED** [review]

**Pull request merged**